### PR TITLE
invite to install and use Thanks

### PR DESCRIPTION
**Pull request by nicolas-grekas**
alternative to https://github.com/symfony/flex/pull/256
sibling to https://github.com/symfony/thanks/pull/28

### CODE REVIEW

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

File: src/Flex.php [hunk 1]
```diff
@@ -416,6 +436,7 @@ public static function getSubscribedEvents(): array
             ScriptEvents::POST_CREATE_PROJECT_CMD => 'configureProject',
             ScriptEvents::POST_INSTALL_CMD => 'install',
             ScriptEvents::POST_UPDATE_CMD => 'update',
+            PluginEvents::COMMAND => 'inspectCommand',
```
Comment (pyrech) [inline src/Flex.php]: @nicolas-grekas looks like v1.0.55 is broken as the `PluginEvents` class does not exist. Did you mean `CommandEvent`?

**Comment by javiereguiluz** [comment]
We could display it randomly one out of 100 times.

**Comment by nicolas-grekas** [comment]
@ciaranmcnulty my proposal: you install the plugin as suggested, then someone fixes https://github.com/symfony/thanks/issues/29 (or in swapped order)

**Comment by nicolas-grekas** [comment]
> displaying every time I run composer

every time composer *update* runs - that's not the same :)

**Comment by ciaranmcnulty** [comment]
It'd be nice if it didn't nag _when I decide not to star the repos_

**Pull request merged**